tests/periph_rtt/Makefile

```diff
@@ -6,3 +6,11 @@ FEATURES_REQUIRED = periph_rtt
 DISABLE_MODULE += periph_init_rtt
 
 include $(RIOTBASE)/Makefile.include
+
+# Put board specific dependencies here
+ifneq (,$(filter-out stm32f1,$(filter stm32%,$(CPU))))
+  # all stm32% but stm32f1 RTT are based on a 16 bit LPTIM, if using the default
+  # 32768KHz configuration TICKS_TO_WAIT will overflow
+  RTT_FREQUENCY ?= 1024
+  CFLAGS += -DRTT_FREQUENCY=$(RTT_FREQUENCY)
```

Comment:
This will break if the board already defined `RTT_FREQUENCY`.

You could use `MOCK_RTT_FREQUENCY`, but that would be a bit of a hack.

Reply:
> This will break if the board already defined RTT_FREQUENCY.
> 
> You could use MOCK_RTT_FREQUENCY, but that would be a bit of a hack.

Yes, that why I filtered to only allow `stm32` CPU's, with this PR they would all use the same configuration that has the `ifndef` logic to allow it.

Reply:
Didn't see the `nucleo-f413zh`, I will address that.